src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react'
+import { Container, Title, Logout, Subtitle } from './styles'
+import { signOut } from '../../service/authentication'
+import PlaylistItem from './PlaylistItem'
+import { playlistsAsyncActions } from '../../store/slices/playlists/slice'
+import { useDispatch } from 'react-redux'
+import { selectCurrentUser } from '../../store/slices/authentication/selectors'
+import { selectOwnPlaylists } from '../../store/slices/playlists/selectors'
+import { useSelector } from 'react-redux'
+import PlaylistData from '../../types/PlaylistData'
+
+const Sidebar = () => {
+
+    const dispatch = useDispatch()
+    const currentUser = useSelector(selectCurrentUser)
+    const ownPlaylists = useSelector(selectOwnPlaylists)
+
+    useEffect(() => {
+        dispatch(playlistsAsyncActions.getCurrentUserPlaylists(currentUser.id))
+    }, [dispatch, currentUser])
+
+    const handleLogout = () => {
+        signOut()
+    }
+    return (
+        <Container>
+            <Title>Collabox</Title>
```

Comment:
It would be good to add a link to home (either with home icon or with the Collabox title). Now we can only return to the main page and to the possibility of adding a new list by a lot of clicking back, or with editing the url.

Reply:
yeah, it should start with that funcionality, fixed